Kitodo/src/test/resources/hibernate.cfg.xml

```diff
@@ -68,7 +68,7 @@
         <mapping class="org.kitodo.data.database.beans.Task"/>
         <mapping class="org.kitodo.data.database.beans.Template"/>
         <mapping class="org.kitodo.data.database.beans.User"/>
-        <mapping class="org.kitodo.data.database.beans.UserGroup"/>
+        <mapping class="org.kitodo.data.database.beans.Role"/>
```

Comment:
Please adjust the ordering of hibernate mapping entries after renaming the class.